docs/pages/docs/docs-theme/start.mdx

```diff
@@ -52,6 +52,22 @@ npm i next react react-dom nextra nextra-theme-docs
   install `nextra` and `nextra-theme-docs` as the add-ons.
 </Callout>
 
+Add the following scripts in `package.json`: 
+
+```json filename="package.json"
+"scripts": {
+  "dev": "next",
+  "build": "next build",
+  "start": "next start"
+},
+```
+
+You can either start the server with, for example if you use npm, `npm run dev`, which will run in development mode or `npm run build && npm run start` for production mode.
+
+<Callout>
+If you're not familiar with next, note that development mode is significantly slower since next compiles every page you navigate to.
```

Comment:
```suggestion
If you're not familiar with Next.js, note that development mode is significantly slower since Next.js compiles every page you navigate to.
```

Am I right, you mean Next.js and not `next` command?